data/trie/branchNode.go

```diff
@@ -549,7 +556,7 @@ func (bn *branchNode) print(writer io.Writer, index int) {
 		}
 		str2 := fmt.Sprintf("+ %d: ", i)
 		_, _ = fmt.Fprint(writer, str2)
-		child.print(writer, index+len(str)-1+len(str2))
+		child.print(writer, index+len(str)-1+len(str2), db)
```

Comment:
Could you extract "index+len(str)-1+len(str2)" into variable with a helpful name

Reply:
Done.